synapse/cortex.py

```diff
@@ -1405,6 +1392,26 @@ async def _addAllLayrRead(self):
         role = await self.auth.getRoleByName('all')
         await role.addRule((True, ('layer', 'read')), gateiden=layriden)
 
+    async def _storLayrFeedDefaults(self):
+
+        for layer in list(self.layers.values()):
+            layrinfo = layer.layrinfo  # type: s_hive.HiveDict
+
+            pushs = layrinfo.get('pushs', {})
+            if pushs:
+                for pdef in pushs.values():
+                    pdef.setdefault('chunk:size', 1_000)
+                    pdef.setdefault('queue:size', 10_000)
```

Comment:
Maybe make these values constants so they don't get out of sync with the values below?